shells/pipes-shell/node/node.js

```diff
@@ -0,0 +1,43 @@
+/*
+* @license
+* Copyright (c) 2019 Google Inc. All rights reserved.
+* This code may only be used under the BSD style license found at
+* http://polymer.github.io/LICENSE.txt
+* Code distributed by Google as part of this project is also
+* subject to an additional IP rights grant found at
+* http://polymer.github.io/PATENTS.txt
+*/
+
+import './config.js';
+
+// optional
+//import '../../lib/pouchdb-support.js';
+//import '../../lib/firebase-support.js';
+//import '../../../node_modules/sourcemapped-stacktrace/dist/sourcemapped-stacktrace.js';
+
+import {Utils} from '../../lib/utils.js';
+import {DeviceApiFactory} from '../device.js';
+
+// usage:
+//
+// ShellApi.observeEntity(`{"type": "address", "name": "East Mumbleton"}`)
+// ShellApi.receiveEntity(`{"type": "com.google.android.apps.maps"}`)
+//
+// ShellApi.receiveEntity(`{"type": "com.music.spotify"}`)
+//
+// results returned via `DeviceClient.foundSuggestions(json)` (if it exists)
+
+global.ShellApi = DeviceApiFactory(`volatile://`, global.DeviceClient);
+
+console.log(`version: feb-27.0`);
+
+// configure arcs environment
+Utils.init(global.envPaths.root, global.envPaths.map);
+
+// test it
+
+setTimeout(() => {
```

Comment:
Maybe comment this out for checkin?

Reply:
Ok. 

Fwiw, I don't expect this version (NodeJs target) of the project works in practice yet. We have yet to run our first Nodejs-on-Android test. I built this mostly so I could compare the dependency trees of Arcs on web vs nodejs, and also just to keep myself honest about making dual-targets work.